src/main/kotlin/org/rust/lang/core/psi/ext/RsItemsOwner.kt

```diff
@@ -100,6 +103,7 @@ val RsItemsOwner.expandedItemsCached: RsCachedItems
                     }
                 }
             }
+            if (it.hasMacroIndex()) ++localMacroIndex
```

Comment:
```suggestion
            if (it.hasMacroIndex()) {
                ++localMacroIndex
            }
```

we omit block only in two cases:
- "exit" statements like `return`, `error`, `throw`, etc.
- `if expr2 else expr2` where both expressions are quite simple